standards/github_folder_template.md

```diff
@@ -0,0 +1,50 @@
+# The .github folder and it's functionality
+
+### What is the .github folder?
+
+The `.github` folder allows users to store GitHub related files/folders. Some of these files and folders perform special functions for organizing your repo, issues, and PRs.
+
+Here are some of those functions:
+- Creating issue templates ([ISSUE_TEMPLATE folder](../.github/ISSUE_TEMPLATE))
+- Creating PR templates 
+- Creating a Code of Conduct
+- Gathering Funding for a project
```

Comment:
I dont think we need to add this, as we dont have any projects that need funding ATM.